File: Milk/bottleOfBeer.cpp
// C++
#include <iostream>
#include <string>
using namespace std;
int main(){
	int x=99;
	string b=" bottles of beer",o=" on the wall",t="Take one down and pass it around, ",f="no more bottles of beer on the wall.";
	while(x>0){
		cout<<x<<(x==1 ? " bottle of beer":b)<<o<<", "<<x--<<(x==0 ? " bottle of beer":b)<<".\n"
			<<t<<(x>0 ? to_string(x)+(x==1 ? " bottle of beer":b):"no more"+b)<<o<<".\n\n";}
	cout<<"No more"<<b<<o<<", no more"<<b<<".\nGo to the store and buy some more, 99"<<b<<o<<".\n";
	return 0;}

File: Milk/bottleOfBeer.py
for x in range(99, 0, -1):
    y = x - 1
    if x == 1:
        print(f"{x} bottle of beer on the wall, {x} bottle of beer.\nTake one down and pass it around, no more bottles of beer on the wall.\n")
    elif y == 1:
        print(f"{x} bottles of beer on the wall, {x} bottles of beer.\nTake one down and pass it around, {y} bottle of beer on the wall.\n")
    else:
        print(f"{x} bottles of beer on the wall, {x} bottles of beer.\nTake one down and pass it around, {y} bottles of beer on the wall.\n")

print("No more bottles of beer on the wall, no more bottles of beer.\nGo to the store and buy some more, 99 bottles of beer on the wall.")
